### PR TITLE
Last Buttercup strings added and translated to FR (french)

### PR DESCRIPTION
**Pull request by fmommeja**
Could you please @perry-mitchell or @sallar review this.  :)

### CODE REVIEW

File: locales/fr/base.json [hunk 1]
```diff
@@ -152,14 +182,14 @@
     "delete": "Supprimer",
     "move-to-group-custom": "Déplacer vers {{title}}",
     "empty-trash": "Vider la corbeille",
-    "empty-trash-question": "Êtes-vous sûr de vouloir vider la corbeille ?",
-    "title-asc": "Titre : Croissant",
-    "title-desc": "Titre : Décroissant"
+    "empty-trash-question": "Êtes-vous sûr de vouloir vider la corbeille ?",
```
Comment (sallar) [inline locales/fr/base.json]: Why is there space added before `:` and `?`?

File: locales/fr/base.json [hunk 2]
```diff
@@ -152,14 +182,14 @@
...
+    "title-asc": "Titre : Croissant",
+    "title-desc": "Titre : Décroissant"
   },
   "intro": {
     "welcome-title": "Bienvenue dans Buttercup.",
     "welcome-back-title": "Bon retour dans Buttercup.",
-    "welcome-caption": "Vous n’avez pas encore ajouté d’archive. Pourquoi ne pas en ajouter une ?",
+    "welcome-caption": "Vous n’avez pas encore ajouté d’archive. Pourquoi ne pas en ajouter une ?",
```
Comment (sallar) [inline locales/fr/base.json]: Also here

File: locales/fr/base.json [hunk 3]
```diff
@@ -178,13 +208,16 @@
     "search": "Chercher"
   },
   "sort": {
-    "title-asc": "Titre : Croissant",
-    "title-desc": "Titre : Décroissant",
-    "time-asc": "Heure : Croissant",
-    "time-desc": "Heure : Décroissant"
+    "title-asc": "Titre : Croissant",
```
Comment (sallar) [inline locales/fr/base.json]: And here

File: locales/fr/base.json (hunk 1 above)
Comment (fmommeja) [reply locales/fr/base.json]: Hi Sallar,

the unbreakable space before double punctuations signs is a typographic rule in French.

Kind regards

File: locales/fr/base.json (hunk 2 above)
Comment (fmommeja) [reply locales/fr/base.json]: Same here

File: locales/fr/base.json (hunk 1 above)
Comment (sallar) [reply locales/fr/base.json]: @fmommeja excuse my ignorance then! thanks :)

Comment (fmommeja) [reply locales/fr/base.json]: @sallar no problem.
I guess rules are specific to each and every language… you can't (neither do I) know everything of it.  ;)

**Pull request merged**